util/mock-server.js

```diff
@@ -81,25 +81,25 @@ app.all('*', (req, res) => {
   if (permanent.has(req.path)) {
     const { status, payload } = permanent.get(req.path);
     res.status(status).json(payload);
-    console.log(`Mocking: ${req.originalUrl}: ${status} ${inspect(payload)}`);
+    // console.log(`Mocking: ${req.originalUrl}: ${status} ${inspect(payload)}`);
```

Comment:
I commented out these logs because they're quite verbose and make it hard to read things

Reply:
maybe we could put these behind a debug flag

Reply:
We might want to add a `logger.debug` method that only runs in dev/test. We already have prior art in `logger.info` `logger.error` and `logger.warn` so could just extend that.

Reply:
Do we need these console logs? They feel like they were useful for development but I don't think they add much benefit going forwards. If there's a use case sometime in the future where a developer needs to know what might be happening somewhere in the code they can add console statements then, this is pretty normal plus that way they can only log the thing they need to know, instead of 1000s of lines

Reply:
They are super useful in development, especially when setting up the tests or when debugging failed tests since it shows us what's being mocked and where.

Reply:
Honestly, I found them not useful at all - I wasn't able to understand why the tests were failing because there was _so much_ stuff being logged out. Hence why I commented them out. 🤷 

I can always just add them back though if there's a general feeling that they're worth keeping?